src/mono/msbuild/android/build/AndroidApp.targets

```diff
@@ -151,6 +151,7 @@
 
     <PropertyGroup>
```

Comment:
@steveisok Does library builder have a dependency on these targets, meaning they will need to go into workload? If so, maybe we should, make the naming better to suite the different scenarios and drop the "App" part in file name + used targets, since it is now not only used to build apps anymore.

Reply:
Yes, I would prefer this be renamed in the workload PR and not in this one,

Reply:
Agree, just wanted to highlight it, do we have a workload PR up yet?

Reply:
I have it in a branch. I'll throw post a PR today.